.github/workflows/doc_deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: page_deploy
+
+on: #push
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy-book:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    # Install dependencies
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: "3.8"
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```

Comment:
Is there a need to install all [requirements](https://github.com/portugueslab/sashimi/blob/master/requirements.txt) from the main program, to build the docs? Is just toml enough maybe? Or not even, in which case `Install dependencies` job can be skipped altogether.

Reply:
You were right! Thanks!